language/English/strings.po

```diff
@@ -267,51 +268,81 @@ msgid "Show now playing info"
 msgstr ""
 
 msgctxt "#31156"
-msgid "[B]Configure Appearance Settings[/B][CR][CR] •     Change the skin[CR] •     Set language and region[CR] •     Change file listing options[CR] •     Set up a screensaver"
+msgid ""
```

Comment:
Whats the reason for this?

Reply:
That was Poedit :) i dont checked the settings after a update. Its normal when you make line breaks in the Translations. I have changed the settings back to not add linebreaks automaticly.